content/docs/use-cases/datasets-cache.md

```diff
@@ -0,0 +1,107 @@
+across environments. Combined storage needs expand rapidly as more people and
+projects acquire the same data, creating duplication and increasing cost. Is
+there an effective way to perform this process?
+
+![](/img/storage-layers.png) _Data storage infrastructure_
+
+DVC's built-in data <abbr>caching</abbr> enables you to implement a simple and
+efficient storage layer globally — for all your team's projects. This approach
+can help in situations such as:
+
+- You need to speed up data transfers from a massive object store on the cloud,
+  or to move a growing data store out of your machine without slowing things
+  down. How do you plug in something in the middle (like a NAS drive) to
+  existing projects?
+
+- Upgrading your storage platform is expensive. Can you pay only for fast access
+  to the data sets that are needed regularly?
+
+- You want to de-duplicate files automatically when multiple people are working
+  on the same datasets (for example, on a
+  [shared development server](#example-shared-development-server)).
+
+- Your team shares access to a GPU server for machine learning
+  [experiments](/doc/user-guide/experiment-management). How can you switch the
+  working datasets quickly, without re-downloading every time?
+
+## How it works
+
+DVC converts your raw data, intermediate artifacts, and final results (e.g.
+machine learning models) into unique storage objects. This optimizes storage by
+avoiding file duplication. <abbr>Cached</abbr> files are mapped back to your
+<abbr>workspace</abbr> using [file links] to minimize transfers.
+
+> 📖 See [Dataset Optimization](/doc/user-guide/large-dataset-optimization) for
+> more details.
+
+[file links]:
+  /doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache
+
+To unify data storage across multiple projects, you can configure a [shared
+cache]. This compounds the deduplication and performance benefits. Having
+separated data storage from other aspects of your projects, you can treat it as
+an independent infrastructure layer, using Ceph,
+[EFS](https://aws.amazon.com/efs/), object storage (e.g. S3,
+[GCS](https://cloud.google.com/storage/), Azure), or any other platform.
+
+![](/img/shared-server.png) _Data storage shared by DVC projects_
+
+To balance speed and cost, you can also use a `dvc remote` (or several) as
+secondary storage. Data only needed in older
+[versions](/doc/use-cases/versioning-data-and-model-files) of your project can
+be kept there. Remote storage is also a great way to back up and
+[share datasets and ML models](/doc/use-cases/sharing-data-and-model-files).
+
+<!-- Advances scenarios? 1. No storage; 2. Only remote -->
+
+## Example: Shared Development Server
+
+Some teams may prefer using a single shared machine to run their experiments.
+This allows better resource utilization, such as GPU access and centralized data
+storage. Each person can work in their own separate <abbr>workspace</abbr> as
```

Comment:
Is this sentence redundant with the info above now?

Reply:
It sort of recaps as an intro to the experiment, yep. Also, mentions "resources" specifically (possibly a keyword).

Reply:
I mean example 😆